src/js/pages/DashboardPage.js

```diff
@@ -171,72 +172,37 @@ var DashboardPage = React.createClass({
+    const resourceLabels = ResourcesUtil.getResourceLabels();
     var data = this.internalStorage_get();
 
     return (
       <Page title="Dashboard">
         <Page.Header breadcrumbs={<DashboardBreadcrumbs />} />
         <div className="panel-grid row">
-          <div className={columnClasses}>
-            <Panel
-              className="dashboard-panel dashboard-panel-chart dashboard-panel-chart-timeseries panel"
-              heading={this.getHeading("DASHBOARD.PANEL_HEADING.CPU")}
-            >
-              <ResourceTimeSeriesChart
-                colorIndex={0}
-                usedResourcesStates={data.usedResourcesStates}
-                usedResources={data.usedResources}
-                totalResources={data.totalResources}
-                mode="cpus"
-                refreshRate={Config.getRefreshRate()}
-              />
-            </Panel>
-          </div>
-          <div className={columnClasses}>
-            <Panel
-              className="dashboard-panel dashboard-panel-chart dashboard-panel-chart-timeseries panel"
-              heading={this.getHeading("DASHBOARD.PANEL_HEADING.MEMORY")}
-            >
-              <ResourceTimeSeriesChart
-                colorIndex={0}
-                usedResourcesStates={data.usedResourcesStates}
-                usedResources={data.usedResources}
-                totalResources={data.totalResources}
-                mode="mem"
-                refreshRate={Config.getRefreshRate()}
-              />
-            </Panel>
-          </div>
-          <div className={columnClasses}>
-            <Panel
-              className="dashboard-panel dashboard-panel-chart dashboard-panel-chart-timeseries panel"
-              heading={this.getHeading("DASHBOARD.PANEL_HEADING.DISK")}
-            >
-              <ResourceTimeSeriesChart
-                colorIndex={0}
-                usedResourcesStates={data.usedResourcesStates}
-                usedResources={data.usedResources}
-                totalResources={data.totalResources}
-                mode="disk"
-                refreshRate={Config.getRefreshRate()}
-              />
-            </Panel>
-          </div>
-          <div className={columnClasses}>
-            <Panel
-              className="dashboard-panel dashboard-panel-chart dashboard-panel-chart-timeseries panel"
-              heading={this.getHeading("DASHBOARD.PANEL_HEADING.GPU")}
-            >
-              <ResourceTimeSeriesChart
-                colorIndex={0}
-                usedResourcesStates={data.usedResourcesStates}
-                usedResources={data.usedResources}
-                totalResources={data.totalResources}
-                mode="gpus"
-                refreshRate={Config.getRefreshRate()}
-              />
-            </Panel>
-          </div>
+          {ResourcesUtil.getDefaultResources().map(resource => {
+            const headingTrnString = `DASHBOARD.PANEL_HEADING.${resourceLabels[
```

Comment:
`Trn`?

Reply:
Force of habit. I'm used to calling these "TRNs" because that's they were called "TRN keys" in the last translation service I used. Happy to change to something clearer